firedrake/mesh.py

```diff
@@ -2046,7 +2055,7 @@ def Mesh(meshfile, **kwargs):
 
 
 @PETSc.Log.EventDecorator("CreateExtMesh")
-def ExtrudedMesh(mesh, layers, layer_height=None, extrusion_type='uniform', kernel=None, gdim=None, name=None):
+def ExtrudedMesh(mesh, layers, layer_height=None, extrusion_type='uniform', periodic=False, kernel=None, gdim=None, name=None):
```

Comment:
Would be useful to have `TorusMesh` and `AnnulusMesh` functions in `utility_meshes.py`.

Reply:
Added `AnnulusMesh` and `SolidTorusMesh`.